src/dev/cordova.md

```diff
@@ -0,0 +1,110 @@
+# How to create a mobile application with Cordova
+
+## Install and setup cordova
+
+[Cordova is a tool](http://blog.ionic.io/what-is-cordova-phonegap/) to build apk (Android) and iOS application from web pages.
```

Comment:
How about "to build Android and iOS applications"? No need to put the file format first I think.